src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala

```diff
@@ -120,6 +120,8 @@ class JsonRpcController(
       handle[SendTransactionRequest, SendTransactionResponse](personalService.sendTransaction, req)
     case req @ JsonRpcRequest(_, "eth_call", _, _) =>
       handle[CallRequest, CallResponse](ethService.call, req)
+    case req @ JsonRpcRequest(_, "eth_estimateGas", _, _) =>
+      handle[CallRequest, EstimateGasResponse](ethService.estimateGas, req)
```

Comment:
Well.....[the reference](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_estimategas) doesn't state that :disappointed: